internal/results/archiver.go

```diff
@@ -60,25 +66,248 @@ func CompressResultsArtifacts(outputDir string, filePaths []string) error {
 
 		tarHeader, err := getFileTarHeader(file)
 		if err != nil {
-			return err
+			return "", err
 		}
 
 		err = tarWriter.WriteHeader(tarHeader)
 		if err != nil {
-			return fmt.Errorf("failed to write tar header for %s: %v", file, err)
+			return "", fmt.Errorf("failed to write tar header for %s: %v", file, err)
 		}
 
 		f, err := os.Open(file)
 		if err != nil {
-			return fmt.Errorf("failed to open file %s: %v", file, err)
+			return "", fmt.Errorf("failed to open file %s: %v", file, err)
 		}
 
 		if _, err = io.Copy(tarWriter, f); err != nil {
-			return fmt.Errorf("failed to tar file %s: %v", file, err)
+			return "", fmt.Errorf("failed to tar file %s: %v", file, err)
 		}
 
 		f.Close()
 	}
 
+	// Create fully qualified path to the zip file
+	zipFilePath, err = filepath.Abs(zipFilePath)
+	if err != nil {
+		return "", fmt.Errorf("failed to get absolute path for %s: %v", zipFilePath, err)
+	}
+
+	// Return the entire path to the zip file
+	return zipFilePath, nil
+}
+
+func createFormField(w *multipart.Writer, field, value string) error {
```

Comment:
What about moving all these funcs to a new file like "internal/results/rhconnect.go"?

Reply:
Good point.  I started it out in archiver but that would make more sense.  I'll move them.